packages/key-did-resolver/README.md

```diff
@@ -6,6 +6,253 @@
 ```
 $ npm install key-did-resolver
 ```
+### Usage
+This code includes support for the curves Ed25519, Secp256k1, Secp256r1 (P-256), Secp384r1 (P-384), and Secp521r1 (P-521) which follow the test vectors at:
+[https://github.com/w3c-ccg/did-method-key/tree/main/test-vectors](https://github.com/w3c-ccg/did-method-key/tree/main/test-vectors)
+
+This code has been tested with the following `did:key`[^1] providers:
+|  Curve              | Repositry                                                      |
+| ------------------- | -------------------------------------------------------------- |
+| Ed25519             | https://github.com/ceramicnetwork/key-did-provider-ed25519     |
+| Secp256k1           | https://github.com/ceramicnetwork/key-did-provider-secp256k1   |
+| P-256, P-384, P-521 | https://github.com/bshambaugh/did-key-creator                  |
```

Comment:
Random thought. This may need to live over in https://github.com/ceramicstudio .